### PR TITLE
[4.0] script.php tweaks

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Remove useless parentheses

### Testing Instructions

Code review

### CODE REVIEW

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -6572,7 +6572,7 @@ private function dropUtf8ConversionTable()
 	 *
 	 * @since   4.0.0
 	 */
-	public function postflight($action, $installer)
+	public function postflight($action)
```
Comment (Quy) [inline administrator/components/com_admin/script.php]: Please remove line 6569.

File: administrator/components/com_admin/script.php [hunk 2]
```diff
@@ -6565,14 +6565,13 @@ private function dropUtf8ConversionTable()
...
-	 * @param   string     $action     Which action is happening (install|uninstall|discover_install|update)
-	 * @param   Installer  $installer  The class calling this method
+	 * @param   string   $action  Which action is happening (install|uninstall|discover_install|update)
 	 *
 	 * @return  boolean  True on success
 	 *
 	 * @since   4.0.0
 	 */
-	public function postflight($action, $installer)
+	public function postflight($action)
```
Comment (joomdonation) [inline administrator/components/com_admin/script.php]: Please don't make change. Although the variable is not used, it is passed to the method when this installer script is called at https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php#L907

In short, your first change (Remove useless parenthese) is valid, the second is not needed (or wrong)

Comment (C-Lodder) [reply administrator/components/com_admin/script.php]: The instance you're referring to is defined here: https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php#L718

Unless maybe I'm missing something

Comment (joomdonation) [reply administrator/components/com_admin/script.php]: It is defined here https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php#L744-L746

Comment (C-Lodder) [reply administrator/components/com_admin/script.php]: Yes, that's the `$manifestClass`, but I'm referring to the `$installer` instance. It's not a global variable or class property. As far as I can see inside `postflight`, it's not being used or passed anywhere

Comment (joomdonation) [reply administrator/components/com_admin/script.php]: It is an argument/parameter of **postflight** method in this class. It is passed when this method is called from outside (exactly on this line of code https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php#L907 ), thus it should not be removed.

I don't know how to explain it more clear, sorry. But I can say that removing it is wrong.

Comment (C-Lodder) [reply administrator/components/com_admin/script.php]: I'll take you word for it then

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on b31837217a73c37eaea39fbaa9067acca7e25b6f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32615">issues.joomla.org/tracker/joomla-cms/32615</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b31837217a73c37eaea39fbaa9067acca7e25b6f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32615">issues.joomla.org/tracker/joomla-cms/32615</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32615">issues.joomla.org/tracker/joomla-cms/32615</a>.</sub>

**Comment by Quy** [comment]
Thanks.

**Pull request merged**